runtime/version/version.go

```diff
@@ -59,7 +59,7 @@ const (
 var version string
```

Comment:
Why is version a var instead of const?

Reply:
I believe @mwhittaker had to do it this way so that the compiler doesn't mark the
definition as unused and delete it from the read-only data.